learning/web/src/components/Layout/index.js

```diff
@@ -132,19 +143,22 @@ class Layout extends React.Component {
   componentDidMount() {
     // If cookies are allowed and they haven't already been initialised,
     // initialise Google Analytics and Tag Manager.
-    if (window.localStorage.getItem("showCookieBanner") === "false" && !Layout.cookiesInitialized) {
+    if (
+      window.localStorage.getItem("showCookieBanner") === "false" &&
+      !Layout.cookiesInitialized
+    ) {
       if (process.env.GATSBY_GOOGLE_TAG_MANAGER_ID) {
         TagManager.initialize({
           gtmId: process.env.GATSBY_GOOGLE_TAG_MANAGER_ID,
-          dataLayerName: 'dataLayer',
-        });
+          dataLayerName: "dataLayer",
```

Comment:
Did the dataLayerName not get  assigned "dataLayer" due to the previous single quotes?

Reply:
See earlier comment - either are fine, but `"` are preferable :)